covidanalyzer.py

```diff
@@ -0,0 +1,120 @@
+import csv
+
+
+def total_country_cases_deaths(total_country,
+                               total_cases,
+                               total_deaths,
+                               total_recovered_cases):
+    with open('covid_cases_stats.csv') as file_obj:
+        reader_obj = csv.DictReader(file_obj)
+        for row in reader_obj:
+            if row['country'] != 'Total:':
+                if row['country']:
+                    total_country += 1
```

Comment:
It will return the total number of rows as country count

Reply:
Resolved..